compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -78,6 +78,10 @@ static inline string MethodPropertiesGetterName(const MethodDescriptor* method)
   return MixedLower("get_" + method->name() + "_method");
 }
 
+static inline string MethodPropertiesGetterHelper(const MethodDescriptor* method) {
```

Comment:
s/Helper/HelperName/ to match the others.

Reply:
Done